DevelopGitHub/GitHubExtension/GitHubExtension.LocalizationTool/Translate/JsonHelper.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Text;
+using GitHubExtension.LocalizationTool.Model;
+using GitHubExtension.LocalizationTool.ViewModel;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace GitHubExtension.LocalizationTool.Translate
+{
+    public class JsonHelper
+    {
+        private const string DoubleQuote = "\"";
+
+        private const string CloseBrackets = "}}";
+
+        private const string �olon = ":";
+
+        private const string Comma = ",";
+
+        private const string FirstOpenQuote = "{\"";
+
+        private const string SecondOpenQuote = "\":{";
+
+        private readonly ObservableCollection<TranslationDataRow> _translationData;
+
+        public JsonHelper(ObservableCollection<TranslationDataRow> translationData)
```

Comment:
Helper should not couple with specific class. It should contains only static dependencies.